src/emoji.ts

```diff
@@ -27,6 +27,8 @@ export interface IEmoji {
     unicode: string;
     skins?: Omit<IEmoji, "shortcodes" | "tags">[]; // Currently unused
     emoticon?: string | string[];
+    customLabel?: string;
+    customComponent?: JSX.Element;
```

Comment:
Comments would be helpful, the other fields are just the types out of Emojibase which could even be inferred